### PR TITLE
Enable arithmetic_paramse for goldilocks64

### PR DESCRIPTION
**Pull request by akokoshn**
Closes https://github.com/NilFoundation/crypto3/issues/154
```
constexpr static const std::size_t s = 0x20;
constexpr static const integral_type arithmetic_generator = 0x01;
constexpr static const integral_type geometric_generator = 0x02;
constexpr static const integral_type multiplicative_generator = 0x07;
constexpr static const integral_type root_of_unity = 0x185629DCDA58878C_cppui64;
```

### CODE REVIEW

**Review by martun — APPROVED** [review]

File: include/nil/crypto3/algebra/fields/arithmetic_params/goldilocks64.hpp [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                private:
+                    typedef params<goldilocks64_base_field> policy_type;
+
+                public:
+                    typedef typename policy_type::modular_type modular_type;
+                    typedef typename policy_type::integral_type integral_type;
+
+                    constexpr static const std::size_t s = 0x20;
+                    constexpr static const integral_type arithmetic_generator = 0x01;
+                    constexpr static const integral_type geometric_generator = 0x02;
```
Comment (ETatuzova) [inline include/nil/crypto3/algebra/fields/arithmetic_params/goldilocks64.hpp:50]: Ok. Not used in placeholder but can be used for geometric_domain. Tests passed successfully

File: include/nil/crypto3/algebra/fields/arithmetic_params/goldilocks64.hpp [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    constexpr static const integral_type multiplicative_generator = 0x05;
+                    constexpr static const integral_type root_of_unity =
+                        0x100000000_cppui64;
+                    constexpr static const integral_type nqr = 0x05;
```
Comment (ETatuzova) [inline include/nil/crypto3/algebra/fields/arithmetic_params/goldilocks64.hpp:54]: Not used, let's remove

File: include/nil/crypto3/algebra/fields/arithmetic_params/goldilocks64.hpp [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                constexpr typename arithmetic_params<goldilocks64_base_field>::integral_type const
+                    arithmetic_params<goldilocks64_base_field>::arithmetic_generator;
+
+                constexpr typename arithmetic_params<goldilocks64_base_field>::integral_type const
+                    arithmetic_params<goldilocks64_base_field>::geometric_generator;
+
+                constexpr typename arithmetic_params<goldilocks64_base_field>::integral_type const
+                    arithmetic_params<goldilocks64_base_field>::multiplicative_generator;
+
+                constexpr typename arithmetic_params<goldilocks64_base_field>::integral_type const
```
Comment (ETatuzova) [inline include/nil/crypto3/algebra/fields/arithmetic_params/goldilocks64.hpp:71]: Not used. Let's remove

**Review by ETatuzova — CHANGES_REQUESTED** [review]

File: include/nil/crypto3/algebra/fields/arithmetic_params/goldilocks64.hpp [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                    typedef params<goldilocks64_base_field> policy_type;
+
+                public:
+                    typedef typename policy_type::modular_type modular_type;
+                    typedef typename policy_type::integral_type integral_type;
+
+                    constexpr static const std::size_t s = 0x20;
+                    constexpr static const integral_type arithmetic_generator = 0x01;
+                    constexpr static const integral_type geometric_generator = 0x02;
+                    constexpr static const integral_type multiplicative_generator = 0x05;
```
[the remark this answers is not in the document]
Comment (akokoshn) [reply include/nil/crypto3/algebra/fields/arithmetic_params/goldilocks64.hpp]: done

**Review by ETatuzova — APPROVED** [review]

**Pull request merged**